charts/kube-prometheus-stack/templates/deprecation.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.checkDeprecation }}
```

Comment:
Please link

`https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack#from-62x-to-63x`